src/ledger/LedgerTxn.cpp

```diff
@@ -1674,21 +1674,23 @@ LedgerTxn::Impl::getRestoredLiveBucketListKeys() const
 }
 
 LedgerKeySet
-LedgerTxn::getAllTTLKeysWithoutSealing() const
+LedgerTxn::getAllKeysWithoutSealing() const
 {
-    return getImpl()->getAllTTLKeysWithoutSealing();
+    return getImpl()->getAllKeysWithoutSealing();
 }
 
 LedgerKeySet
-LedgerTxn::Impl::getAllTTLKeysWithoutSealing() const
+LedgerTxn::Impl::getAllKeysWithoutSealing() const
 {
-    abortIfWrongThread("getAllTTLKeysWithoutSealing");
+    abortIfWrongThread("getAllKeysWithoutSealing");
     throwIfNotExactConsistency();
     LedgerKeySet result;
+    // Subtle: iterating over mEntry implies iterating over *modified* entries
+    // in the given ltx, so callers assume entries were modified for eviction
```

Comment:
Nit: "callers assume entries were modified for eviction purposes" sounds a little odd as written, as we explicitly don't want to evict modified entries and this sounds like we use the ltx to evict the modified entries or something.